### ISSUE TITLE
Error while pulling docker image

### ISSUE THREAD

**Issue by anirudt** [noonien/docker-openwrt-buildroot#12]
Hi, the following error message propped up as I used the following command, after running `docker search.`
Error:

```
FATA[0006] Tag latest not found in repository noonien/openwrt-buildroot
```

Is there any tag you need to update?

**Comment by noonien** [comment]
The current available tags are `12.09` and `14.07`.

I'm not sure if the code still works or what needs to be updated to make it work.

There have been 2 major versions since this repo was maintained and i'm not sure that they build in the same way.

**Comment by noonien** [comment]
I've updated the image with the 17.01 and 18.06 versions, they are tagged, there's also the `latest` tag now which points to 18.06.
